include/openssl/ssl.h

```diff
@@ -282,6 +282,9 @@ typedef int (*SSL_custom_ext_parse_cb_ex) (SSL *s, unsigned int ext_type,
 /* Typedef for verification callback */
 typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
 
+/* In TLSv1.3 allow a non-(ec)dhe based kex_mode */
+# define SSL_OP_ALLOW_NO_DHE_KEX                         0x00000001U
```

Comment:
Without exactly answering your question, I do not believe it was a violation of our ABI stability guarantee to change SSL_OP_ALL in this fashion.
That is, we guarantee forward stability but not backwards -- something compiled against 1.1.0f is not expected to work when using a 1.1.0e shared library at runtime.